### ISSUE TITLE
[0.9.0 staging-1772] Civics: exception with trigger selection

### ISSUE THREAD

**Issue by beastyn** [StrangeLoopGames/EcoIssues#18074]
1) Create a law
2) Choose a trigger
Get this
```
[02:19:21.379] Exception in view notifySystem.NullReferenceException: Object reference not set to an instance of an object.
  at Eco.Shared.View.PropertySubscription.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.View.View.NotifyChanged (System.String propname) [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.View.ViewManager.ReceiveViewUpdate (Eco.Shared.Serialization.BSONArray updates) [0x00000] in <00000000000000000000000000000000>:0 
  at ClientPacketHandler.ReceiveWhile (Eco.Shared.Networking.NetworkClient client, TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at NetworkManager.OnTimeLimitedUpdate (TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.EntityQueryBuilder+F_E.Invoke (Unity.Entities.Entity entity) [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.PlannerGroup.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.FramePlannerSystem.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystemGroup.UpdateAllSystems () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.Font+FontTextureRebuildCallback.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
```

**Comment by johnkslg** [comment]
Not seeing any exceptions, can you try again and give more specific instructions?

**Comment by beastyn** [comment]
On build version its pretty simple
1. /civics spawngov
2. Press revise on a generated law
3. Change trigger

**Comment by SlayksWood** [comment]
It works like this for me:
- start to create law.
- select trigger 'didn't vote' (any trigger except chat sent)
- select trigger 'chat sent'
- select trigger 'didn't vote' (any trigger except chat sent)
- I have exception:
```
Exception in view notifySystem.NullReferenceException: Object reference not set to an instance of an object.
  at Eco.Shared.View.PropertySubscription.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.View.View.NotifyChanged (System.String propname) [0x00000] in <00000000000000000000000000000000>:0 
  at Eco.Shared.View.ViewManager.ReceiveViewUpdate (Eco.Shared.Serialization.BSONArray updates) [0x00000] in <00000000000000000000000000000000>:0 
  at ClientPacketHandler.ReceiveWhile (Eco.Shared.Networking.NetworkClient client, TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at NetworkManager.OnTimeLimitedUpdate (TimeLimit timeLimit) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.CullingGroup+StateChanged.Invoke (UnityEngine.CullingGroupEvent sphere) [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.PlannerGroup.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at FramePlanner.FramePlannerSystem.OnUpdate () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystemGroup.UpdateAllSystems () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.ComponentSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
  at System.Runtime.Remoting.Contexts.CrossContextDelegate.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
```
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5282076/Player.log)

**Comment by mirasrael** [comment]
Probably fixed. Need to check.

**Comment by IcedForge** [comment]
Keeps happening when you switch between multiple triggers in the law interface, sometimes it starts to cause these exceptions and once they start firing off you need to restart client / reconnect to stop it.

**Comment by SlayksWood** [comment]
Seems the the steps like with other issues:
- open economy viewer
- open court, draft a law. Select any trigger, select another trigger, I have exception.

**Comment by mirasrael** [comment]
It wasn't related to not unsubscribed subscriptions... but was related to unsubscribed subscriptions :)
Issue happened when you had multiple subscriptions for same property for same view which then was removed in change callback for this property. We had logic which protects from removing subscription in change callback, but it only worked for single property. I fixed this to work with any number of unsubscriptions.
